### PR TITLE
Update the press release

### PR DESCRIPTION
**Pull request by Redstone-ray**
There's some bad coding practise here and some very mid code, but it is very time sensitive so we should just ship it, I will go back and slowly clean up the code after.

Changes:
- Added press release
- Added Borealis rocket to the rockets list
- Updated competition

- - -
This change is [Reviewable](https://reviewable.io/reviews/waterloo-rocketry/website-react/76)

### CODE REVIEW

File: src/routes/Competition.jsx [hunk 1]
```diff
@@ -17,33 +17,33 @@ const Competition = () => {
...
-                propulsion and structures to avionics, recovery, and payload.
-                Overuse of commercial components is discouraged. In this way, the
-                IREC promotes student design and a spirit of simple, holistic
-                design: excess complexity is the bane of the student rocket
-                engineering team.
+                The Launch Canada Challenge is Canada’s
+                first-ever major national rocketry competition.
+                It brings together Canada’s passionate and
+                rapidly growing amateur and student rocketry community
+                with academia, industry and government to facilitate
```
Comment (slightlyskepticalpotat) [inline src/routes/Competition.jsx:26]: nit: academia, industry, and government _partners_ sounds more natural

File: src/routes/Competition.jsx [hunk 2]
```diff
@@ -17,33 +17,33 @@ const Competition = () => {
...
+                the development of national talent in the interdisciplinary
+                field of rocket engineering, while fostering a strong
+                culture of safety.
+              </p>
+              <p>
+                The team have competed in the 2023 Launch Canada challenge
```
Comment (slightlyskepticalpotat) [inline src/routes/Competition.jsx:32]: nit: since the team is acting as a whole here, it should be "the team has competed"

**Review by slightlyskepticalpotat — COMMENTED** [review]
The top of the competition and press release images are cut off, is this intentional?
image
image

**Review by slightlyskepticalpotat — APPROVED** [review]
approving to not block merge

**Comment by Redstone-ray** [comment]
We don't have pictures of the correct scale, if you change the size of your browser then it will be correct, I think we'll need to move to multiple adaptive images if we really want to fix that.

**Pull request merged**